Exec/reacting_tests/bubble_convergence/GNUmakefile

```diff
@@ -7,7 +7,7 @@ COMP	         = gnu
 
 USE_MPI          = TRUE
 USE_GRAV         = TRUE
-USE_REACT        = TRUE
+USE_REACT        = FALSE
```

Comment:
why was this changed?

Reply:
That was also changed in a9e7943d74, probably by accident since it was a large commit.

Pushing a change to revert this now ...

Reply:
Done in db1be2b6a0af99a96433c59fec178a2b85ba2c64